docker/cypress-E2E-tests-postgres.yml

```diff
@@ -0,0 +1,73 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Cypress Setup Postgres
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    postgresql:
+        extends:
+            file: ./postgresql.yml
+            service: postgresql
+
+    artemis-app:
+        extends:
+            file: ./artemis.yml
+            service: artemis-app
+        depends_on:
+            postgresql:
+                condition: service_healthy
+        env_file:
+            - ./artemis/config/cypress-application-local.env
+            - ./artemis/config/cypress-application-postgres.env
+        environment:
+            SPRING_PROFILES_ACTIVE: "artemis,bamboo,bitbucket,jira,prod,docker"
+
+    nginx:
+        extends:
+            file: ./nginx.yml
+            service: nginx
+        # the artemis-app service needs to be started, otherwise there are problems with name resolution in docker
+        depends_on:
+            artemis-app:
+                condition: service_started
+        volumes:
+            - ./nginx/artemis-nginx-cypress.conf:/etc/nginx/conf.d/artemis-nginx-cypress.conf:ro
+        ports:
+            - "80:80"
+            - "443:443"
+            # see comments in artemis/config/cypress-application-local.env why this port is necessary
+            - "54321:54321"
+
+    artemis-cypress:
```

Comment:
I would create an ``artemis-cypress`` base service as for ``artemis-app`` this way you can leave out the command and the environment variables and just have to change them in one location.

Reply:
Base service is looking good except that the ``depends_on:`` has to be inside of the children and can't be extended according to: https://docs.docker.com/compose/compose-file/#extends